index.js

```diff
@@ -20,11 +23,11 @@ LESSCompiler.prototype.compile = function(params, callback) {
   var data = params.data;
   var path = params.path;
 
-  less.render(data, {
+  var options = _.extend({}, this.options, {
```

Comment:
Historically, in Brunch and its core plugins we avoid importing monolithic utility libs and do the `Object.keys().forEach` kind of thing often. But I agree that strategy could be revisited.

How about something more micro and targeted, like `util-extend`?